### DISCUSSION TITLE
[2052410]魔法使之夜

### DISCUSSION THREAD

**Discussion by user2Xp** [pjy612/SteamManifestCache#150] [Share]
Appid:2052410
```
2052411;6ab2d901d86a9f777078ad2d3310ba844676045d51fce0e67d2ad6ec7bb3c39d
```

[2052411_3234638570754145427.zip](https://github.com/pjy612/SteamManifestCache/files/13761478/2052411_3234638570754145427.zip)

![9](https://github.com/pjy612/SteamManifestCache/assets/154731439/c228420e-3101-41ca-ba0d-44731c147422)
<img width="604" alt="8" src="https://github.com/pjy612/SteamManifestCache/assets/154731439/37645409-231e-42a5-8856-dd4c7e170874">

email:2655271197@qq.com
code:B9E9D683235727C0

**Comment by pjy612** [comment]
没code么？

**Reply by user2Xp** [reply]
有了

**Reply by pjy612** [reply]
查收

**Reply by miku9964** [reply]
我焯，碟，我就尝试一下，没想到真有啊，谢谢

**Comment by gonghaocheng-chia** [comment]
大佬，我那个战争雷霆不行吗？
